aspnetcore/signalr/streaming/samples/3.0/Startup.cs

```diff
@@ -0,0 +1,65 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using SignalRChat.Hubs;
+
+namespace SignalRChat
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.Configure<CookiePolicyOptions>(options =>
+            {
+                options.CheckConsentNeeded = context => true;
+                options.MinimumSameSitePolicy = SameSiteMode.None;
+            });
+
+
+            services.AddCors(options => options.AddPolicy("CorsPolicy",
+            builder =>
+            {
+                builder.AllowAnyMethod().AllowAnyHeader()
+                       .WithOrigins("http://localhost:44317")
+                       .AllowCredentials();
+            }));
+
+            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_3_0);
```

Comment:
I didn't think the compat version was necessary for major releases ... just minors. Is that incorrect?

Reply:
I have no idea! This was probably created with `dotnet new`

Reply:
I looked it up.

> The default compatibility for ASP.NET Core 2.1 and later 2.x apps that do not call SetCompatibilityVersion is 2.0 compatibility. 

It defaults to the major release, but it's fine tho. All good.

https://docs.microsoft.com/aspnet/core/mvc/compatibility-version